new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -580,6 +589,48 @@ public static boolean buildAppArchive(AppParams params) {
             .build();
   }
 
+  /**
+   * Archive an application from provided ear or war file that can be used by WebLogic Image Tool
+   * to create an image with the application for a model-in-image use case.
+   *
+   * @param srcFile full path to ear or war application file to archive
+   * @return true if the operation succeeds
+   */
+  public static boolean archiveApp(
```

Comment:
TestActions class should not contain the implementation, it should call impl class.

Reply:
fixed